Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/types.go

```diff
@@ -2590,6 +2590,11 @@ type SecurityContextConstraints struct {
 	unversioned.TypeMeta `json:",inline"`
 	ObjectMeta           `json:"metadata,omitempty"`
 
+	// SortPriority influences the sort order of SCCs when evaluating which SCCs to try first for
+	// a given pod request based on access in the Users and Groups fields.  The higher the int, the
+	// higher priority.  If scores for multiple SCCs are equal they will be sorted by name.
+	SortPriority int `json:"sortPriority" description:"influences the sort order when evaluating SCCs available to the user"`
```

Comment:
"determines which SCC is used when multiple SCCs allow a particular pod; higher priority SCCs are preferred"